src/Gui/GLPainter.h

```diff
@@ -24,12 +24,6 @@
 #ifndef GUI_GLPAINTER_H
 #define GUI_GLPAINTER_H
 
-#ifdef FC_OS_WIN32
-#ifndef NOMINMAX
-#define NOMINMAX
-#endif
-#include <windows.h>
-#endif
```

Comment:
I wonder if this is why it fails on windows. I'm quite sure gl.h depends on windows.h.
Worth noting is that win32 just means any windows, not just 32 bit versions.